stable/rabbitmq-ha/templates/configmap.yaml

```diff
@@ -73,9 +73,11 @@ data:
     cluster_formation.node_cleanup.only_log_warning = true
     cluster_partition_handling = {{ .Values.rabbitmqClusterPartitionHandling }}
 
+    {{- if eq .Values.rabbitmqUsername "guest" }}
```

Comment:
It's unfortunate the chart's default user is guest, we really should change that at some point.